html_test.go

```diff
@@ -131,6 +131,35 @@ func TestHtmlReplaceSelector(t *testing.T) {
 		})
 	}
 }
+
+func TestHtmlTokenizeToMaps(t *testing.T) {
+	const html = `
+		<html>
+			<body class="main">
+				<h1>Title</h1>
+				<p>First paragraph.</p>
+				<p class="content">Second paragraph.</p>
+				<div class="footer">Footer</div>
+		</html>
+	`
+	var maps []map[string]string
+	for _, m := range htmlTokenizeToMaps(html) {
+		if m["Type"] == "Text" && strings.TrimSpace(m["Data"]) == "" {
+			continue
+		}
+		maps = append(maps, m)
+	}
```

Comment:
Replace the `const html` above with a short one-liner `<div>` snippet (with no whitespace between tags) to avoid this awkward filtering process.